soapui/tools/chocolateyInstall.ps1

```diff
@@ -1,25 +1,35 @@
-$ErrorActionPreference = 'Stop';
+$ErrorActionPreference = 'Stop'
 $Version = "5.7.0"
 $toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
-$url        = "http://dl.eviware.com/soapuios/$Version/SoapUI-x32-$Version.exe"
 $url64      = "http://dl.eviware.com/soapuios/$Version/SoapUI-x64-$Version.exe"
 
+
+
 $packageArgs = @{
   packageName   = $env:ChocolateyPackageName
   unzipLocation = $toolsDir
   fileType      = 'EXE'
-  url           = $url
   url64bit      = $url64
-
   softwareName  = 'soapui*'
-
-  checksum      = '6f32e0f5415e759afcd88bbd19786d16263a36584d38373a54650298e53f84a0'
-  checksumType  = 'sha256'
   checksum64    = '86B6E5658886A0876D0196394F91A470E1AF7D62D3067EEB94C1F8C5293D3B18'
   checksumType64= 'sha256'
-
   silentArgs    = "-q"
   validExitCodes= @(0, 3010, 1641)
 }
 
+$regKey = Get-UninstallRegistryKey  -SoftwareName $packageArgs['softwareName'] 
```

Comment:
Just realised that SoapUI installer, doesn't uninstall previous versions.
So logic will need tweaked handle multiple previous 32bit installs.
Guess that ... `Select-Object-First 1 -ExpandProperty PSParentPath ` would do it?

Reply:
I think you're trying to solve a non-issue here, what are the chances people installed the x86 version on an x64 system?

Reply:
Also, the place is quite peculiar since by aborting the installers leaves you in an "uninstalled"  state.

Reply:
> Also, the place is quite peculiar since by aborting the installers leaves you in an "uninstalled" state.

Hi there,

Thanks very much for looking into this and accepting my PR.

Apologies if I'm misunderstanding here,  but the error was raised before the call to `Install-ChocolateyPackage` so on a 32 bit OS `upgrade` fails, but leaves existing application and Chocolatey package in place. 

I think that was my testing showed, if you look at the output from Chocolatey above?

Somewhat restricted by not having 32 bit Windows to test on.